ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -0,0 +1,153 @@
+        """
+        Navigate to OperatorHub Page
+
+        """
+        logger.info("Click On Operators Tab")
+        self.choose_expanded_mode(
+            mode=True, by_locator=deployment["Operators Tab"], type=By.XPATH
+        )
+        logger.info("Click On OperatorHub Tab")
+        self.do_click(deployment["OperatorHub Tab"], type=By.LINK_TEXT)
+
+    def navigate_installed_operators(self):
+        """
+        Navigate to Installed Operators page
+
+        """
+        logger.info("Click On Installed Operators Tab")
+        self.choose_expanded_mode(
+            mode=True, by_locator=deployment["Operators Tab"], type=By.XPATH
+        )
+        logger.info("Click On OperatorHub Tab")
+        self.do_click(deployment["Installed Operators Tab"], type=By.LINK_TEXT)
+
+    def install_ocs_opeartor(self):
+        """
+        Install OCS Opeartor
+
+        """
+        self.navigate_operatorhub()
+
+        logger.info("Search OCS Operator")
+        self.do_send_keys(
+            deployment["Search Operators"],
+            text="OpenShift Container Storage",
+            type=By.CSS_SELECTOR,
+        )
+
+        logger.info("Choose OCS")
+        self.do_click(deployment["Choose OCS"])
+
+        logger.info("Click Install OCS")
+        self.do_click(deployment["Click Install OCS"], type=By.CSS_SELECTOR)
+
+    def install_storage_cluster(self):
+        """
+        Install Storage Cluster
+
+        """
+        self.navigate_installed_operators()
+
+        logger.info("Click On OCS Installed")
+        self.do_click(deployment["OCS Installed"], type=By.CSS_SELECTOR)
+
+        logger.info("Storage Cluster Tab")
+        self.do_click(deployment["Storage Cluster Tab"], type=By.CSS_SELECTOR)
+
+        logger.info("Click On 'Create Storage Cluster'")
+        self.do_click(deployment["Create Storage Cluster"], type=By.CSS_SELECTOR)
+
+        if self.mode == "internal":
+            self.install_internal_cluster()
+        elif self.mode == "lso":
+            self.install_lso_cluster()
+
+    def install_internal_cluster(self):
+        """
+        Install Dynamic Cluster
+
+        """
+        logger.info("Click On 'Internal'")
+        self.do_click(deployment["Internal"], type=By.XPATH)
+
+        logger.info("Select Storage Class")
+        self.do_click(deployment["Storage Class Dropdown"], type=By.CSS_SELECTOR)
+        self.do_click(deployment["thin"], type=By.CSS_SELECTOR)
+
+        logger.info("Choose OSD Size")
+        self.do_click(deployment["OSD Size Dropdown"], type=By.CSS_SELECTOR)
+        self.do_click(deployment[self.osd_size], type=By.XPATH)
+
+        if self.is_encryption:
+            self.do_click(deployment["Enable Encryption"], type=By.XPATH)
+
+        logger.info("Create Storage Cluster Page")
+        self.do_click(deployment["Create Storage Cluster Page"], type=By.XPATH)
+
+    def install_lso_cluster(self):
+        """
+        Install LSO Cluster
+
+        """
+        pass
```

Comment:
If LSO cluster installation is not implemented, it might be better to raise some exception instead of silently do nothing (and pretend that cluster was installed).